stable/distribution/README.md

```diff
@@ -259,6 +259,7 @@ The following table lists the configurable parameters of the distribution chart
 | `postgresql.affinity`                           | PostgreSQL node affinity                                               | `{}`                                                               |
 | `postgresql.tolerations`                        | PostgreSQL node tolerations                                            | `[]`                                                               |
 | `redis.password`                                | Redis password                                                         | ` `                                                                |
+| `redis.existingSecret`                          | Redis existingSecret for password                                      | ` `                                                                |
```

Comment:
I think we should keep the convention we have in other charts, which is `secretName`

Reply:
@danielezer I took the convention as per upstream chart 
https://github.com/bitnami/charts/blob/master/bitnami/redis/values.yaml#L205

Reply:
ok, makes sense